### ISSUE TITLE
Keyboard shortcuts stop working until Spectacle is restarted

### ISSUE THREAD

**Issue by ivanbrennan** [eczarny/spectacle#744]
When my computer has been inactive for a while, I often find that Spectacle's keyboard shortcuts have stopped working.

For example, if Chrome is focused, ⌥⌘→ will select the next browser tab instead of making the window occupy the right half of the screen.

Selecting window configurations from the Spectacle menu works, but I need to quit Spectacle and reopen it to get keyboard shortcuts working again.

Though it happens frequently, I'm not entirely sure how to repro. Putting the computer to sleep and immediately waking it doesn't seem to be sufficient. I suspect it's happening either after long periods of inactivity/sleep, or when a long time has passed since Spectacle was last opened.

I'm using a 2013 MacBook Air, running macOS 10.12.3, but I experienced the same issue in Mavericks and Yosemite. I'm running Spectacle 1.2

**Comment by arlemi** [comment]
Happened to me as well, a restart fixed it... wasn't able to reproduce either.

**Comment by hay** [comment]
Yup, i have the same problem as well (i'm running El Cap on an early 2015 Macbook Pro). Funnily enough i don't actually need to restart Spectacle, just re-running the app again from Spotlight (which gives the preferences window) also seems to work.

**Comment by air** [comment]
Thanks @hay, will try that next time.

**Comment by llamerr** [comment]
Have this problem with new macOS 11.0.1 (20B50), not sure when this started to happen, I did not use Spectacle for last half year. Though for me restarting an app doesn't help. Some shortcuts just not working at all. Those that have default actions in active app?

Move to the right half — ⌥⌘→

switches tab instead of Spectacle shortcut, however

Move to the upper right — ⌃⌘→

works fine
